src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: 200px;
+  height: 408px;
+  margin-right: 48px;
```

Comment:
I see you apply margins for giving a space between cards. 
It's a bit hard to maintain. 
I recommend you use such rule as "gap" for parent element with a class `.main__card`

Read, it's easy) [link](https://developer.mozilla.org/en-US/docs/Web/CSS/gap)